solution/src/main/kotlin/com/cosmotech/solution/service/SolutionServiceImpl.kt

```diff
@@ -271,6 +281,24 @@ internal class SolutionServiceImpl(
       hasChanged = true
     }
 
+    if (solution.security != null && existingSolution.security == null) {
```

Comment:
remove and prevent modification of security. security modification should be done in endpoint security